api/utils/rights.js

```diff
@@ -548,15 +571,15 @@ function dbLoadEventsData(params, apps, callback) {
         callback(null, params.member.eventList, params.member.viewList);
     }
     else {
-        async.map(apps, getEvents, function(err, events) {
+        getEvents({ appIds, appNamesById }, function(err, events) {
             var eventList = {};
             for (let i = 0; i < events.length; i++) {
```

Comment:
I don''t think we need this  for cycle anymore. Before it was async map, so you got:
[result array1, result array2......]
But now all results are in single array at root. 
The same for views